pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -101,14 +107,16 @@ typedef struct  s_ndarray
     int32_t                 buffer_size;
     /* True if the array does not own the data */
     bool                    is_view;
+    /* stores the order of the array: order_f or order_c */
+    enum e_order            order;
```

Comment:
I think adding this is a very good idea, but as mentioned above, we need to make sure we agree on how `f_order` arrays are handled as if this flag is used we need to be sure that the handling is consistent